README.md

```diff
@@ -6,61 +6,45 @@ The current set of extensions can be read on [GitHub Pages](https://ocfl.github.
 
 See also [pending pull requests](https://github.com/OCFL/extensions/pulls) for extensions under discussion.
 
-## Organization of this repository
+## Specifying Community Extensions
 
-Community extensions should be written as GitHub flavored markdown files in the `docs` directory of this repository. The
+### Layout
+
+Community extensions MUST be written as GitHub flavored markdown files in the `docs` directory of this repository. The
 filename of an extension is based on its *Registered Name* with a `.md` extension. 
 
 Extensions are numbered sequentially, and the *Registered Name* of an extension is prefixed with this 4-digit, zero-padded
-decimal number. The *Registered Name* should be descriptive, use hyphens to separate words and have a maximum of 250 
-characters in total. New, or substantially revised, extensions MUST use the next available number based on extensions current
-at the time of merging. The *Registered Name* MUST be listed in the header of the extension file. 
+decimal number. The *Registered Name* should be descriptive, use hyphens to separate words, and have a maximum of 250
+characters in total. New extensions MUST use the next available prefix number that's available at the time of merging.
+
+Extensions are intended to be mostly static once published. Substantial revisions of content beyond simple fixes warrants publishing a new extension, and marking the old extension obsolete by updating the *Obsoletes/Obsoleted by* sections in each extension respectively.
 
-An example/template is available in this repository as [OCFL Community Extension](docs/0000-example-extension.md) and is rendered
+An example/template is available [here](docs/0000-example-extension.md) and is rendered
```

Comment:
Need to fix `here` to have good anchor text

Reply:
This can wait as a minor later fix

Reply:
@zimeon : I believe this has been addressed:
https://github.com/OCFL/extensions/pull/32/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22